2.js

```diff
@@ -0,0 +1,86 @@
+const readline = require('readline')
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+})
+
+const isValidTile = character => {
+  if (typeof character !== 'string') return false
+
+  const characterCode = character.charCodeAt(0)
+
+  if (
+    (characterCode >= 65 && characterCode <= 90)
+      || characterCode === 95
+  ) {
+    return true
+  } else {
+    return false
+  }
+}
+
+const tilesRemaining = {
+  A: 9,
+  B: 2,
+  C: 2,
+  D: 4,
+  E: 12,
+  F: 2,
+  G: 3,
+  H: 2,
+  I: 9,
+  J: 1,
+  K: 1,
+  L: 4,
+  M: 2,
+  N: 6,
+  O: 8,
+  P: 2,
+  Q: 1,
+  R: 6,
+  S: 4,
+  T: 6,
+  U: 4,
+  V: 2,
+  W: 2,
+  X: 1,
+  Y: 2,
+  Z: 1,
+  _: 2
+}
+
+console.log('\nScrabble Tiles Reamining Calculator\n\nEnter uppercase characters representing the tiles that have been played.\n\'_\' represents a blank tile.\nExample: AERETOXMYCNS_B\n(type exit to exit)\n')
```

Comment:
Consider using multiple lines to output this in order to increase readability.  You could use ES6 string templates (using the backtick) to include new lines in the string itself, as well, but I'd probably either define the output message elsewhere, or use multiple `console.log`s. (I despise horizontal scrolling.)